openmls/src/messages/tests/test_proposals.rs

```diff
@@ -2,305 +2,10 @@ use openmls_rust_crypto::OpenMlsRustCrypto;
-        };
-
-        let proposal_add_alice1 = Proposal::Add(add_proposal_alice1);
-        let proposal_reference_add_alice1 =
-            ProposalReference::from_proposal(ciphersuite, &crypto, &proposal_add_alice1).unwrap();
-        let proposal_add_bob1 = Proposal::Add(add_proposal_bob1);
-
-        // Frame proposals in MlsPlaintext
-        let mls_plaintext_add_alice1 = MlsPlaintext::new_proposal(
-            framing_parameters,
-            LeafIndex::from(0u32),
-            proposal_add_alice1.clone(),
-            &alice_credential_bundle,
-            &group_context,
-            &MembershipKey::from_secret(Secret::random(
-                ciphersuite,
-                &crypto,
-                None, /* MLS version */
-            )),
-            &crypto,
-        )
-        .expect("Could not create proposal.");
-        let mls_plaintext_add_bob1 = MlsPlaintext::new_proposal(
-            framing_parameters,
-            LeafIndex::from(1u32),
-            proposal_add_bob1.clone(),
-            &alice_credential_bundle,
-            &group_context,
-            &MembershipKey::from_secret(Secret::random(
-                ciphersuite,
-                &crypto,
-                None, /* MLS version */
-            )),
-            &crypto,
-        )
-        .expect("Could not create proposal.");
-
-        // This should set the order of the proposals.
-        let proposals = &[&mls_plaintext_add_alice1, &mls_plaintext_add_bob1];
-
-        let proposal_queue =
-            ProposalQueue::from_proposals_by_reference(ciphersuite, &crypto, proposals);
-
-        // Now let's iterate over the queue. This should be in order.
-        let proposal_collection: Vec<&QueuedProposal> =
-            proposal_queue.filtered_by_type(ProposalType::Add).collect();
-
-        assert_eq!(proposal_collection[0].proposal(), &proposal_add_alice1);
-        assert_eq!(proposal_collection[1].proposal(), &proposal_add_bob1);
-
-        let proposal_or_refs = &[
-            ProposalOrRef::Proposal(proposal_add_bob1.clone()),
-            ProposalOrRef::Reference(proposal_reference_add_alice1),
-        ];
-
-        let sender = Sender {
-            sender_type: SenderType::Member,
-            sender: LeafIndex::from(0u32),
-        };
-
-        // And the same should go for proposal queues built from committed
-        // proposals. The order here should be dictated by the proposals passed
-        // as ProposalOrRefs.
-        let proposal_queue = ProposalQueue::from_committed_proposals(
-            ciphersuite,
-            &crypto,
-            proposal_or_refs,
-            proposals,
-            sender,
-        )
-        .unwrap();
-
-        let proposal_collection: Vec<&QueuedProposal> =
-            proposal_queue.filtered_by_type(ProposalType::Add).collect();
-
-        assert_eq!(proposal_collection[0].proposal(), &proposal_add_bob1);
-        assert_eq!(proposal_collection[1].proposal(), &proposal_add_alice1);
-    }
-}
-
```

Comment:
I think the last test below here is already in the other `test_proposals.rs`, so we should be able to get rid of this file altogether.

Reply:
Good point! Since it's a codec test, I think it should remain in this file. I deleted it in the new test file.